app/page.tsx

```diff
@@ -1,7 +1,14 @@
-export default function Home() {
+import discogs from "./utils/SearchApi";
+
+export default async function Home() {
+  const result = await discogs.search();
+
   return (
-    <main>
-      <h1>Home Page</h1>
+    <main className="flex flex-col items-center justify-center mt-60">
+      <h1 className="font-bold text-6xl">Home Page</h1>
+      <h1 className="font-bold text-6xl">
```

Comment:
you should only ever have one h1 on a page at a time

Reply:
For sure. I was only doing this because I couldn't get console log in vs code to work. Removed.